src/Conversions.cc

```diff
@@ -1390,6 +1390,12 @@ msgs::ParticleEmitter ignition::gazebo::convert(const sdf::ParticleEmitter &_in)
     header->add_value(_in.Topic());
   }
 
+  // todo(anyone) add particle_scatter_ratio field in particle_emitter.proto
```

Comment:
I asked this on another PR too, but just to be thorough: let's not forget to add this field in Fortress

Reply:
Added field in https://github.com/ignitionrobotics/ign-msgs/pull/162